content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,173 @@
+description: Learn how to use Quality Gates in your pipeline.
+is_beta: true
+further_reading:
+- link: "/continuous_integration/tests"
+  tag: "Documentation"
+  text: "Learn about Test Visibility"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="http://dtdg.co/code-quality" header="false" >}}
+Quality Gates are in private beta. To request access, complete the form.
+{{< /callout >}}
+
+Quality Gates allows you to gate your workflows based on signals in Datadog. You can create two types of rules: Static Analysis and Tests.
+For example, you can block code from being merged if it introduces new [flaky tests][8] or code style violations; issues that wouldn't normally fail your CI/CD pipelines and end up deployed to production.
+
+Using Quality Gates, you have control over what is merged into the default branch, and ultimately on what is deployed.
+This allows you to ensure that the code running in production is meeting high quality standards, reducing incidents and unwanted behaviors.
+
+## Set up Quality Gates
+
+There are two main steps required to set up and use Quality Gates:
+1. Create one or more rules in Datadog's UI.
+2. Invoke Quality Gates by using the `datadog-ci gate evaluate` command in your CI pipeline.
+
+## Create a rule
+
+To create Quality Gates rules for your organization, your user account must have the `quality_gate_rules_write` [permission][1].
+
+1. In Datadog, navigate to [**CI** > **Quality Gate Rules**][2] and click **+ New Rule**.
+2. Select the rule type. You can choose between `Static Analysis` and `Test`.
+3. Define the rule scope. The rule scope defines when the rule should be evaluated. For example, you can create a rule that it is evaluated only on specific repositories and branches. To define the scope for a rule, switch to `select when to evaluate` and add included or excluded values for the scope. More information on [rule scopes](#rule-scope).
+4. Define the rule condition. The rule condition states in which scenario the rule fails, failing the related pipeline (if the rule is blocking). You can select one of the existing rule conditions for the rule type you have selected.
+
+   The following example shows how to create a static analysis rule that will fail when there are one or more static analysis violations with "error" status and "security" category being introduced in a specific commit:
+
+   {{< img src="ci/qg_rule_condition_sa_errors_security.png" alt="Rule for static analysis security errors" style="width:80%;">}}
+
+5. Select whether the rule should block the pipeline or not when it fails. Non-blocking rules are still evaluated, but
+they do not block the pipeline when they fail.
+6. Select the time window over which the query will be evaluated.
+7. Select a rule name that describes the rule that you are creating.
```

Comment:
Should we add any info here for the rule name? e.g. the rule name will be shown in the CI pipeline logs and GH Checks

Reply:
As for now we only have logs I don't think it makes much sense to add it 🤔 Although later it will be shown basically everywhere, from checks to rule evaluations